snippets/language-go.cson

```diff
@@ -107,3 +107,6 @@
   'goroutine function':
     'prefix': 'gf'
     'body': 'go ${1:func}($0)'
+  'test':
```

Comment:
Would be nice if we can change this to 'test function' as this is the description you'll see in the autocomplete popup.